proto/desmos/profiles/v1beta1/genesis.proto

```diff
@@ -30,4 +30,9 @@ message GenesisState {
     (gogoproto.nullable) = false,
     (gogoproto.moretags) = "yaml:\"params\""
   ];
+
+  string ibc_port_id = 5 [
+    (gogoproto.moretags) = "yaml:\"ibc_port_id\"",
+    (gogoproto.customname) = "IBCPortID"
```

Comment:
@bragaz That's not correct. The `customname` is ignored only if used along with the `grpc-gateaway` Protobuf plugin, which is used only when generating the querier code. It can (and should) still be used elsewhere. Please @dadamu do not remove this line.